charts/kargo/values.yaml

```diff
@@ -337,6 +337,11 @@ controller:
   ## @param controller.podAnnotations Optional annotations to add to pods. Merges with `global.podAnnotations`, allowing you to override or add to the global annotations.
   podAnnotations: {}
 
+  ## All settings relating to the service account for the controller
+  serviceAccount:
+    ## @param controller.serviceAccount.annotations Optional annotations to add to the service account.
```

Comment:
I think it may be worthwhile mentioning in the comment that this is the appropriate place to include any annotations required by IRSA.